contrib/scalar/scalar.txt

```diff
@@ -35,6 +35,16 @@ The command implements various subcommands, and different options depending
 on the subcommand. With the exception of `clone` and `list`, all subcommands
 expect to be run in an enlistment.
 
+The following options can be specified _before_ the subcommand:
```

Comment:
We should be consistent of the use of `command`/`subcommand`/`verb` throughout the series. I think you've used `command` before, which seems most correct.

Reply:
Actually, I start by detailing that "The command" (by which I mean `scalar`) implements "various subcommands". So I think I'll settle on "subcommand".